Feint

Apricite opened the pull request:
## About the PR
Adds NT TE-Feint Station, a team effort made with help of Arenaconspiracy, Robinthedragon, Freelancin and BlueVibes.

## Why / Balance
"We need to port more shit. Hopefully Feint"
"Damnit, we really need more lowpop maps..."
"so more maps when"

## Technical details
Map yml, prototype .yml and evac shuttle .yml added. I also edited two cs files just like Arenaconspiracy did in https://github.com/RonRonstation/ronstation/pull/307. The yml were easy, but CS is dark magic and I have no idea if it will work. 

## Media
<img width="4832" height="4672" alt="Feint" src="https://github.com/user-attachments/assets/cd9ba275-19c4-4846-a646-959b8ea6ef25" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- add: Added Feint station

Apricite committed:
Feintsweep

Apricite committed:
what just happened...

Apricite commented:
Oop, seems like Freelanin readded half-tile decals, so I will give it a quick update

Apricite committed:
Merge branch 'carpmosia:dev' into feint

Apricite committed:
Map Updated +attributions

Atakku commented:
Ye it was part of oasis PR

Apricite committed:
Invalid error fix (?)

arenaconspiracy commented on Content.IntegrationTests/Tests/PostMapInitTest.cs:
```diff
@@ -104,6 +104,7 @@ public sealed class PostMapInitTest
             "Relic",
             "dm01-entryway",
             "Exo",
+			"Feint"
```
```suggestion
			"Feint" // Carpmosia-edit - Feint
```

arenaconspiracy commented on Resources/map_attributions.txt:
```diff
@@ -63,3 +63,8 @@
+
+// Carpmosia-start - Start of modifications
+- files: ["feint.yml"]
+  authors: Apricity, Arenaconspiracy, Robinthedragon, Freelancin, Bluevibes
```
```suggestion
  authors: Apricity, Arenaconspiracy, Robinthedragon2, Freelancin, Bluevibes
```
Also txt files don't have comment syntax by virtue of. Uh. Not being code. But that's whatever.

Atakku replied:
They are in yaml format tbh

arenaconspiracy reviewed (changes requested):
I'm not sure who handled what on this revision, so I repeatedly use the singular "you" when suggesting changes be made. This is probably the harshest I've been in a map review, particularly about Security. It pains me to be so harsh about it but is up there with the fundamental flaws All Of Saltern has (these are different flaws)
I was in different states of mind at different points in this review, with the cutoffs probably being visible at points.
I'm sorry for being late with this review, and addressing all of it will probably take uncomfortably long. The dream of carpmaps however remains.
Also there's some comment cleanup to be done.

Citations follow.

Security has fuckall map beacons :(
Also the clinic could get a beacon while we're at it 
<img width="258" height="275" alt="tmp" src="https://github.com/user-attachments/assets/83bf407d-4dff-4f91-bcc9-89eac6639c9c" />

This bit with the 2 directional windows at the engineering front desk should just be solid glass 
<img width="208" height="389" alt="tmp" src="https://github.com/user-attachments/assets/22a5083d-f5a9-4760-ab2c-c30da81047a5" />

The clinic losing maints access is Disasterous. You can swap the backway & maints airlocks and connect the clinic to the maints west, leaving a little alcove 
<img width="928" height="793" alt="image" src="https://github.com/user-attachments/assets/639c3b8b-141e-4da7-bb22-f6a940a42c92" />

Who invited this guy 
<img width="487" height="445" alt="tmp" src="https://github.com/user-attachments/assets/49decab4-e06e-4024-b568-fe809b988ce4" />

Tinted glass is actually just as weak as basic glass (Oversight that honestly can be fixed.... in another PR? It's worth doing that instead of replacing it here) and fireplaces aren't meant to be rotated. I'd rather lose the fireplace for a power or atmos monitor console, maybe squeeze other consoles in the office too
<img width="433" height="317" alt="tmp" src="https://github.com/user-attachments/assets/1f1e9107-7555-411f-ad73-a1e8b6599d36" />

My first thoughts entering this room were "this is remarkably poorly decorated" which is a bit mean. It could benefit with replacing the dark maint floors with plating, with the flatpack crate in the middle - and the fuel crate west of the controller 
<img width="409" height="582" alt="tmp" src="https://github.com/user-attachments/assets/9ce3901a-e664-44cb-8755-d96e94f8696d" />

Instead of scattered boards, you could put a surplus board crate (migrated to totes now I believe?) and put the materials on a rack. Otherwise I like this bit a lot
<img width="394" height="370" alt="tmp" src="https://github.com/user-attachments/assets/5b6b2478-e65c-4c5e-a41e-996712029b0d" />

This cold storage would unequivocally benefit from having the generators already as structures, and the other flatpacks on racks instead of in crates. The airlock could also be made into a 4 directional rosetta? To connect all 4 directions for movement convenience
<img width="524" height="640" alt="tmp" src="https://github.com/user-attachments/assets/ffefc0df-ead8-4fc2-bfc4-b3f7436932cd" />

I love this 
<img width="254" height="253" alt="tmp" src="https://github.com/user-attachments/assets/574bedf4-56f3-4667-b294-46113dcea63d" />

East of the PA could probably fit a power monitor and a few closets, maybe a suit storage unit. I kept old Feint engineering very bare to my own dismay because of the space constraints, now that there is more space you shouldn't be afraid to use it 
<img width="377" height="494" alt="tmp" src="https://github.com/user-attachments/assets/c361f3b3-25c4-44cf-84e1-942bc9cf9f87" />

I'm not particularly sure why this needs to start bolted. And the button isn't actually linked to anything 
<img width="486" height="228" alt="tmp" src="https://github.com/user-attachments/assets/9de20b3d-6cff-46e1-9fa8-dba7f0004351" />

The containment substation & SMES are unnamed 
<img width="861" height="459" alt="tmp" src="https://github.com/user-attachments/assets/06e8dd9d-0578-48d1-9b9f-abc5741f70d4" />

This button in atmos doesn't have a frame and isn't linked to anything 
<img width="407" height="521" alt="tmp" src="https://github.com/user-attachments/assets/253e3fff-25c6-416c-80f4-5f902e71e488" />

That one IS linked but doesn't have a frame 
<img width="474" height="288" alt="tmp" src="https://github.com/user-attachments/assets/0a39b5c6-72eb-4d5f-a76b-17c7afcf5cd3" />

Also opening the ex-maints door in the clinic stucks it open permanently until you pry the barricade 
<img width="540" height="323" alt="tmp" src="https://github.com/user-attachments/assets/194f30f4-e58d-423d-8679-1c07e3cb3840" />

This corner on the table should just be a wall 
<img width="316" height="273" alt="tmp" src="https://github.com/user-attachments/assets/4a2ec634-f19b-4b72-91fc-b912962bb260" />

I forgot I never fixed this console's name in the AI core 
<img width="465" height="340" alt="tmp" src="https://github.com/user-attachments/assets/e38bb2f2-6547-4f1d-b2cd-cc2d5d877620" />

The boxer room gas pipes were never connected to anything 
<img width="971" height="528" alt="tmp" src="https://github.com/user-attachments/assets/fac9659c-3f26-4ace-a0e4-eeb1f4a1d973" />

Also it never actually occurred to me that the AI external airlock leads to a completely gated off area, you can't get to or from any other external airlock without cutting down grilles or fences or breaking a window 
<img width="991" height="553" alt="tmp" src="https://github.com/user-attachments/assets/608ee463-951b-4aae-b03a-b150952c22a0" />

This part of evac is ominously unlit 
<img width="631" height="410" alt="tmp" src="https://github.com/user-attachments/assets/3365834f-05f3-41bb-96e5-2a366ea29420" />

So is this part just south of it 
<img width="571" height="712" alt="tmp" src="https://github.com/user-attachments/assets/37810c08-92d8-462c-a3a4-799bc66ed73d" />

This corner table should also be a wall 
<img width="532" height="318" alt="tmp" src="https://github.com/user-attachments/assets/e2565af4-65c6-46e5-8585-00a0bcd15041" />

The internal airlocks don't bolt eachother. Those logic gates are also not necessary on 1 wide doorways.
I'm not sure why this whole setup is necessary, why the side maints door is left as an assembly, but that's probably besides the point
<img width="775" height="412" alt="tmp" src="https://github.com/user-attachments/assets/ee240062-1aed-4def-ae40-6075a7303f78" />

This maints tunnel south of security feels barren and unfinished, and I will dare guess that it's actually unfinished? Also that south substation is unnamed 
<img width="1201" height="712" alt="tmp" src="https://github.com/user-attachments/assets/efedc29c-1e6e-440c-a7b1-e247748e51e6" />

This airlock assembly isn't cycling. Also the outer airlock should probably have a small patch of lattice you can walk off onto
<img width="764" height="600" alt="tmp" src="https://github.com/user-attachments/assets/f83fef8d-423b-4a4f-bac0-663a13db65db" />

This APC isn't connected to a substation 
<img width="910" height="522" alt="tmp" src="https://github.com/user-attachments/assets/1660267d-72e0-4132-b329-cdf73248a611" />

This genpop window doesn't have cable beneath the grille 
<img width="419" height="275" alt="tmp" src="https://github.com/user-attachments/assets/a88dbc3c-4137-418a-a618-d9871ecbbd81" />

The turnstiles have no decals. And the entry turnstile could stand to be an airlock instead. Brigmedical should have an airlock instead of 2 windoors, and use Security access
<img width="725" height="338" alt="tmp" src="https://github.com/user-attachments/assets/48c5932c-e103-4a3c-832c-13351103d3fa" />

One of these medibeds should be a rollerbed instead. Remove the advanced kit & keep only the combat kit. Add a cell recharger
<img width="372" height="305" alt="tmp" src="https://github.com/user-attachments/assets/e418ca7a-4f3b-4595-bb18-f13780069b4c" />

Absolutely fucking not! Lose the locker. Security does not deserve all that, ESPECIALLY not a renamed paramedic voidsuit, with 2! paramedics on the map already width="364" height="429" alt="tmp" src="https://github.com/user-attachments/assets/28ecee06-fdd9-4c28-90ed-fb1e12961033" />

No floor tile under brigmedical door. Honestly brigmedical being adjacent to a public hallway already feels sus from a roundflow perspective
<img width="237" height="315" alt="tmp" src="https://github.com/user-attachments/assets/5d0c3d7d-073b-4a8d-8b20-02008ffcd651" />

Overall security feels really, really damn big for no good reason. It Does Not Need A 3 Tile Desk. The warden's room is twice as big as it needs to be, twice as big as it has Things to have inside. The table clutter struggles to fill the actual tables. The locker room is miserably empty, holding a mere 4 officer lockers while being larger than Plasma station's, which holds like 12. (Also that sectech is unpowered. Also I'm not sure why security needs a radiation suit more than they need a 2nd bio suit) Genpop lockers is awkwardly squished to shoehorn in a wildly overstocked brigmedical. HoS' room is awkwardly stretching thin what little it needs. As a sidenote, Security front typically uses brig access to let Command & the Lawyer enter, to congregate for whatever, for HoS to meet command, Brig access' name and what it's Actually Used For are unintuitively inversed so I don't blame you for inverting it.
Feint is objectively, unequivocally a Small Map. All the departments and small, and the areas between them are small. It's strikingly obvious that this revision of security was made by someone excited to make security. It fails to stay cohesive with the rest of the map, growing to the size of Plasma or Oasis Security which ends up hurting the entire map. I'm very sorry for being so thoroughly scalding with this, I know how much work this is, but I can't allow, like, any of this. Security will have to be ripped up a third time before I accept this into carpmos. This revision of Security doesn't need to be half as big as it is right now; genpop is 2 empty spaces, one of which could be halved the other of which could be removed entirely. The armory's shell and surroundings are almost as big as the canteen when they Do Not Need To Be.
I'm sorry but this is just wrong.
<img width="697" height="735" alt="tmp" src="https://github.com/user-attachments/assets/420d914e-8378-4867-98d6-ccf858a14b32" />

There does not need to be a secfab outside the armory and an ammofab inside. The ammofab shouldn't even be mapped on stations 
<img width="321" height="154" alt="tmp" src="https://github.com/user-attachments/assets/99b436a4-c5bd-47b0-a30d-53163df18f1c" />

The security substation isn't connected to HV cables, and you have to walk through All Of Security to access it 
<img width="241" height="401" alt="tmp" src="https://github.com/user-attachments/assets/52480223-c5da-4438-a3a0-f2f61c05c604" />

The leftmost heater in atmos is awkwardly hanging off the wall, could stand to be removed 
<img width="347" height="338" alt="tmp" src="https://github.com/user-attachments/assets/0ae141e0-24c5-4bc4-9201-36228e68212f" />

Who invited this guy 
<img width="576" height="341" alt="tmp" src="https://github.com/user-attachments/assets/04558af7-d405-4913-93e9-aabb75c04933" />

I like the cleanup done to the canteen 
<img width="1092" height="586" alt="tmp" src="https://github.com/user-attachments/assets/8ab1b06e-9607-44b3-9a6c-666906608d16" />

Who invited this guy 
<img width="241" height="249" alt="tmp" src="https://github.com/user-attachments/assets/f36fd46d-5b4a-4b8f-86a3-20fb057f5e12" />

It's a good thing you put a window above these because I forgot to suggest it earlier 
<img width="343" height="460" alt="tmp" src="https://github.com/user-attachments/assets/55aed07f-53f8-4a28-a39a-e1987f859f71" />

The medical maints external airlock should probably have lattice added outside the outer airlock, I forgot to add it when I made that 
<img width="500" height="402" alt="tmp" src="https://github.com/user-attachments/assets/1854f523-ecd9-4673-9f1b-9621445d47b5" />

Chapel maints access doesn't have tiles under it 
<img width="345" height="396" alt="tmp" src="https://github.com/user-attachments/assets/24936a63-50e3-4924-b7ab-18c90ba5ae30" />

The railings, benches, and confessional in the chapel could probably stand to be rearranged like this 
<img width="765" height="596" alt="tmp" src="https://github.com/user-attachments/assets/fe7e0ce1-c009-44d7-962d-1f3475e996ef" />

The station doesn't deserve a second pinpointer in the vault. Especially since carpmos doesn't have BSO 
<img width="326" height="521" alt="tmp" src="https://github.com/user-attachments/assets/a2acb003-7019-438a-8ae8-5da94fa58d6b" />

Basic plasma & uranium windows don't actually have any more health than basic glass 
<img width="621" height="242" alt="tmp" src="https://github.com/user-attachments/assets/a8bec874-b3ae-40e0-bde8-641fd4f8f98b" />

Interrogator lamp's flashing is not tied to it's power cell, or I don't know what the thought behind this was 
<img width="196" height="134" alt="tmp" src="https://github.com/user-attachments/assets/8634e8d5-4c98-477c-9dfe-bb5cc478a96b" />

This button isn't actually linked to the reporter shutters 
<img width="496" height="344" alt="tmp" src="https://github.com/user-attachments/assets/71109e65-40df-42f9-bcae-8ab6b391e333" />

The blast door access reader doesn't actually work. Should probably fix that in a separate PR 
<img width="605" height="699" alt="tmp" src="https://github.com/user-attachments/assets/4baabdc1-588a-4872-8d7d-37fa23d35c3a" />

I like this a lot, I like the whole area really
<img width="743" height="717" alt="tmp" src="https://github.com/user-attachments/assets/739935dd-cfd7-4f6a-9ad2-9f2f60a6954a" />

That's all I could note this go around. Again I am sorry to have been so harsh earlier but that's just the way it is. Addressing the minor issues with the rest of the map and bulldozing security again and it's good, probably.
Also the evac shuttle needs a DeviceNetworkComponent added in order for the evac ETA timers to work, I don't have the exact codeblock on hand right now, you can ask Freelancin to do it I think.

Atakku commented on Resources/map_attributions.txt:
```diff
@@ -63,3 +63,8 @@
 - file: ["amber.yml"]
   authors: Southbridge, Flipsie27
+
+// Carpmosia-start - Start of modifications
```
```suggestion
# Carpmosia-start - Start of modifications
```

Atakku commented on Resources/map_attributions.txt:
```diff
@@ -63,3 +63,8 @@
+// Carpmosia-start - Start of modifications
+- files: ["feint.yml"]
+  authors: Apricity, Arenaconspiracy, Robinthedragon, Freelancin, Bluevibes
+// Carpmosia-end - End of modifications
```
```suggestion
# Carpmosia-end - End of modifications
```

Atakku commented on Resources/shuttle_attributions.txt:
```diff
@@ -162,3 +162,8 @@
 - files: ["Shuttles/ShuttleEvent/traveling_china_cuisine.yml"]
   authors: TheShuEd
+
+// Carpmosia-start - Start of modifications
```
```suggestion
# Carpmosia-start - Start of modifications
```

Atakku commented on Resources/shuttle_attributions.txt:
```diff
@@ -162,3 +162,8 @@
+// Carpmosia-start - Start of modifications
+- files: ["Shuttles/emergency_feint.yml"]
+  authors: Arenaconspiracy
+// Carpmosia-end - End of modifications
```
```suggestion
# Carpmosia-end - End of modifications
```

Atakku reviewed (changes requested):
attributions are in YAML format

Atakku committed:
Merge branch 'dev' into feint

Atakku commented on Content.IntegrationTests/Tests/PostMapInitTest.cs:
```diff
@@ -105,6 +105,7 @@ public sealed class PostMapInitTest
             "Relic",
             "dm01-entryway",
             "Exo",
+            "Feint" // Carpmosia-edit - Feint port
```
```suggestion
            "Feint", // Carpmosia-edit - Feint port
```

Atakku commented on Content.Benchmarks/MapLoadBenchmark.cs:
```diff
@@ -47,7 +47,7 @@ public async Task Cleanup()
     }
 
-    public static string[] MapsSource { get; } = { "Empty", "Saltern", "Box", "Bagel", "Dev", "CentComm", "Core", "TestTeg", "Packed", "Omega", "Reach", "Meta", "Marathon", "MeteorArena", "Fland", "Oasis", "Convex"};
+    public static string[] MapsSource { get; } = { "Empty", "Saltern", "Box", "Bagel", "Dev", "CentComm", "Core", "TestTeg", "Packed", "Omega", "Reach", "Meta", "Marathon", "MeteorArena", "Fland", "Oasis", "Convex", "Feint"}; // Carpmosia-edit - Feint
```
```suggestion
    public static string[] MapsSource { get; } = { "Empty", "Saltern", "Box", "Bagel", "Dev", "CentComm", "Core", "TestTeg", "Packed", "Omega", "Reach", "Meta", "Marathon", "MeteorArena", "Fland", "Oasis", "Convex"};
```
this is a benchmark file, we dont rly run any benchmarks yet, so there is no need

Atakku committed:
Apply suggestions from code review

Co-authored-by: arenaconspiracy <113312769+arenaconspiracy@users.noreply.github.com>

Atakku committed:
Delete Carpmosia modification entries

Removed Carpmosia modification section from attributions.

Atakku committed:
Update file path in shuttle_attributions.txt

Atakku committed:
Remove empty line in map_attributions.txt

Atakku committed:
Update file path in map_attributions.txt

Atakku committed:
Update authors format in shuttle_attributions.yml

Atakku committed:
Rename map_attributions.txt to map_attributions.yml

Atakku committed:
Add map attributions for various files

Atakku committed:
Add shuttle attributions with author details

Added attributions for various shuttle files, including authorship information.

Atakku commented:
Unfucked the shit

Atakku commented:
although hold on

Atakku commented:
Also gotta add it to rotation as well

Atakku committed:
trying to bypass the linter

Atakku committed:
Trying to bypass the linter

Atakku committed:
Bypass attribution linter

Atakku committed:
Bypass attribution linter

SnappingOpossum commented:
Kinda late but I'd like to note the directional plasma windows do have a little more health than a normal directional window, still weaker than a reinforced one though since that has a better modifier set.

Apricite committed:
for better or worse, it's done

arenaconspiracy reviewed (changes requested):
I must say, this has done an excellent job of addressing the previous review. Security is Quite Good now and adheres to the rest of the map. From all I could find with my time today, one more patch and this should be mergable I believe. Great work.

Citations follow.

I must have missed this stacked microwave and vending machine in engineering.
Also, the mapped jack-o-lanterns are out of season now, and should be axed. I'll try to refrain from mentioning in the rest of the review
<img width="491" height="426" alt="tmp" src="https://github.com/user-attachments/assets/d4d7171a-2e74-4fa3-8de2-44780c1ae3fc" />

Cold storage only has 4 tesla coils when the containment assembly is made for 6 
<img width="345" height="476" alt="tmp" src="https://github.com/user-attachments/assets/aba96df2-f2f7-4677-a071-68ea7dd16ce0" />

I should have illustrated what I meant by the airlock thing in my last review. Something like this would serve to be much more ergonomic 
<img width="393" height="480" alt="tmp" src="https://github.com/user-attachments/assets/d58e5312-35fd-4325-a8fa-105bafba4e7d" />

Clinic missing tile under north door 
<img width="367" height="260" alt="tmp" src="https://github.com/user-attachments/assets/05ad65ec-3c34-4886-8104-7a16c54fcd21" />

The maints bit should have a dim light & an emergency closet or two, and the divot with the chair should have some sort of light & maybe a fire safety closet 
<img width="355" height="404" alt="tmp" src="https://github.com/user-attachments/assets/2a348402-1a06-4d19-b8ee-081024ea30ab" />


This door in EVA doesn't have power 
<img width="532" height="454" alt="tmp" src="https://github.com/user-attachments/assets/ccaadc9f-ccd0-4cb5-b47c-d4b7712516eb" />

Floating lamp north of clinic 
<img width="562" height="356" alt="tmp" src="https://github.com/user-attachments/assets/b793501b-75ec-4ed0-8cca-b2269e5071e7" />

Lol whoops. That's probably fine 
<img width="450" height="684" alt="tmp" src="https://github.com/user-attachments/assets/b842bc21-b2e0-4ced-bae2-f31ee3c0c336" />

The containment APC isn't actually linked to the MV 
<img width="268" height="185" alt="tmp" src="https://github.com/user-attachments/assets/84213f77-5749-4686-83cb-754ca38e9414" />

Atmos button linked now, still doesn't have a frame though (not that big of a deal.....)
<img width="282" height="235" alt="tmp" src="https://github.com/user-attachments/assets/ad8bb768-f419-4b4f-a6b3-1439d860d383" />

Southwest solars APC is linked to neither LV nor MV 
<img width="339" height="262" alt="tmp" src="https://github.com/user-attachments/assets/41f73491-8458-46bf-bd64-acaf42a85a31" />

Service hallway APC isn't connected to MV 
<img width="567" height="289" alt="tmp" src="https://github.com/user-attachments/assets/db3da5c7-26f8-44e2-a8df-35b4bbb6d708" />

These 2 airlocks are access free when they're supposed to use brig access
<img width="649" height="327" alt="tmp" src="https://github.com/user-attachments/assets/5042ea3b-8bdd-47ba-aa3e-4582b4fd8808" />

The armory implanter crate & sec front contraband crate could stand to swap places 
<img width="494" height="401" alt="tmp" src="https://github.com/user-attachments/assets/ff0e7135-5b14-4020-8c99-374c950fb445" />

Extra advanced kit should be removed, and a cell recharger should be added. Otherwise this is real good 
<img width="388" height="286" alt="tmp" src="https://github.com/user-attachments/assets/e7714c36-f94f-4b84-82ab-b82afd4eb4c8" />

This falls into the trap of "Walk through most of security to put someone in genpop" ron feint did. 
<img width="299" height="497" alt="tmp" src="https://github.com/user-attachments/assets/6f03bbee-f4f7-40f4-b52b-9b8b5874221c" />

Consider the following 
<img width="737" height="604" alt="tmp" src="https://github.com/user-attachments/assets/01547e95-b782-447a-934d-6a23d519fe45" />

The virology directional firelocks should be either removed or moved overlapping with the windoors (I TOTALLY DIDN'T DIE TO THIS ONE TIME AS A LONEOP BACK ON RON) 
<img width="677" height="219" alt="tmp" src="https://github.com/user-attachments/assets/086a2125-c738-4324-bc16-a1e00fe6be8e" />

Addressing all this and any other issues can Probably be left to the players on live. We can hotfix it on carp now.

Apricite committed:
All Things Fixed

Apricite committed:
Fixed invalids, mats and made logic gates work better

Apricite committed:
fixed the radiation collector deffiency error

SnappingOpossum commented:
This should be external access.
<img width="602" height="345" alt="Screenshot 2025-11-05 063004" src="https://github.com/user-attachments/assets/bb836c7d-0997-4d20-b8e4-3247173e0a5a" />

All of these are command access, at least one should be external
<img width="427" height="486" alt="Screenshot 2025-11-05 063102" src="https://github.com/user-attachments/assets/6843ea2e-2d13-401c-a1f8-bcc70fbfafe5" />

SnappingOpossum commented:
Oh and a salvage shuttle console should be mapped.

arenaconspiracy commented:
It has recently come to my attention that upstream has updated their maps to use HoS & warden weapon spawners. The warden spawner will need to be mapped otherwise there will be no energy shotgun in the locker. Albeit I'm not certain if our dev branch has pulled from upstream stable with the spawners yet.

Atakku commented:
> It has recently come to my attention that upstream has updated their maps to use HoS & warden weapon spawners. The warden spawner will need to be mapped otherwise there will be no energy shotgun in the locker. Albeit I'm not certain if our dev branch has pulled from upstream stable with the spawners yet.

We are on latest stable unless they pushed a hotfix or smth

Atakku commented:
> It has recently come to my attention that upstream has updated their maps to use HoS & warden weapon spawners. The warden spawner will need to be mapped otherwise there will be no energy shotgun in the locker. Albeit I'm not certain if our dev branch has pulled from upstream stable with the spawners yet.

I think the PR is so old it needs to update xd

Apricite committed:
this damn console, Im adding it the third time

SnappingOpossum commented:
Yeah that fixes my issues

Atakku commented:
<img width="346" height="580" alt="image" src="https://github.com/user-attachments/assets/cb9ff720-97f6-4339-a567-e7d0f0d51244" />

i think you might have some doors that you accidentally opened

Atakku commented:
specifically:
<img width="599" height="274" alt="image" src="https://github.com/user-attachments/assets/e39adf7d-97cd-4989-a3bb-53b894b826b3" />
<img width="598" height="264" alt="image" src="https://github.com/user-attachments/assets/6e53613a-1572-4e00-aa01-b87e270113fa" />
and some door at 36.5, -61.5
<img width="405" height="234" alt="image" src="https://github.com/user-attachments/assets/abecaf52-b9d4-4760-af8b-a122245ea655" />

Apricite committed:
doors stuck

arenaconspiracy reviewed (changes requested):
HoS gets WT550 magazines but no WT550 
<img width="329" height="434" alt="tmp" src="https://github.com/user-attachments/assets/f82fa4ed-5af4-4f5e-9e97-a5039d3024d8" />

Arrivals needs these things mapped
<img width="922" height="543" alt="tmp" src="https://github.com/user-attachments/assets/32ac836f-2d59-464b-9ea2-734e6d7d1def" />

Apricite committed:
Weapon ready and spawns active

Atakku commented:
<img width="722" height="63" alt="image" src="https://github.com/user-attachments/assets/5970b84a-f783-4513-bdab-af46d9272b5f" />

the test is pretty self explanatory

Apricite committed:
.

wavesneglected commented:
<img width="848" height="650" alt="image" src="https://github.com/user-attachments/assets/88eea273-6cc2-4397-acc1-dc882e8a8dd8" />
<img width="839" height="788" alt="image" src="https://github.com/user-attachments/assets/6fed2849-c8af-4ee7-b948-8fcd73a64570" />
<img width="1081" height="861" alt="image" src="https://github.com/user-attachments/assets/ec56a002-91a1-48ae-9b2c-b75f8f6cdf47" />
<img width="688" height="627" alt="image" src="https://github.com/user-attachments/assets/8753cb6d-3e4b-444c-a3db-62baecdf098d" />
<img width="627" height="560" alt="image" src="https://github.com/user-attachments/assets/46c5d556-597b-4949-b1b2-c9f437b79ee3" />
<img width="612" height="473" alt="image" src="https://github.com/user-attachments/assets/4a8ea7a0-6f99-4148-8952-bf9bf4559c0c" />

These airlocks leading to maintenance are not locked to maintenance, making anyone able to go through them.

wavesneglected commented:
<img width="659" height="360" alt="image" src="https://github.com/user-attachments/assets/b439c51a-ca9c-4577-8bca-78b75048ec8e" />
 
 This button, which i assume was added to control the shutters is non-functional.

Apricite committed:
fixed recent issues

wavesneglected commented:
Besides the doors and that button, I think everything else appears to be alright.

Apricite committed:
Merge branch 'carpmosia:dev' into feint

Apricite committed:
weapon spawners

arenaconspiracy reviewed (changes requested):
\@gulps nervously
<img width="513" height="282" alt="tmp" src="https://github.com/user-attachments/assets/4d440ac3-9b45-49c5-ab91-d19d028f3fd7" />

arenaconspiracy commented:
Logging off for today but if celeste or atakku can review bulldozing the 2 extra magazines if that goes through soon you can dismiss my review & merge.

SnappingOpossum commented:
> Logging off for today but if celeste or atakku can review bulldozing the 2 extra magazines if that goes through soon you can dismiss my review & merge.

Can actually use the HoS weapon spawner now since that got merged.

Apricite committed:
removed extra mags because they pissed me off

arenaconspiracy commented:
That's what I was talking about. The spawner places a WT550 with 2 magazines in addition to the 2 pre-mapped magazines, which is not desired.

arenaconspiracy reviewed (approved):
Fixed.

Atakku reviewed (changes requested):
`fixrotations` reports 7 entities rotated, other than that lgtm

Apricite committed:
rotations fixed + I used variantize command

wavesneglected reviewed (approved):
Everything I've seen that was an issue has been fixed. Approved.

Atakku reviewed (changes requested):
<img width="529" height="527" alt="image" src="https://github.com/user-attachments/assets/86a385c8-198c-4636-ad00-2cd6c38d0230" />

curtains stuck on opening

Apricite committed:
omfg

Atakku reviewed (approved):
lgtm

imatsoup reviewed (approved):
chalk this up to 'i forgot i was one of the people supposed to review this'

arenaconspiracy commented on Resources/Prototypes/_Carpmosia/Maps/feint.yml:
```diff
@@ -0,0 +1,65 @@
+      - type: StationEmergencyShuttle
+        emergencyShuttlePath: /Maps/_Carpmosia/Shuttles/emergency_feint.yml
+      - type: StationJobs
+        availableJobs: # 44 jobs total w/o latejoins & interns, 59 jobs total w/ latejoins & interns
```
```suggestion
        availableJobs: # 43 jobs total w/o latejoins & interns, 58 jobs total w/ latejoins & interns
```
No BSO on carpmos so off by one.

Apricite replied:
Is it incorrect though? I counted them

arenaconspiracy replied:
I counted like 3 times last night to make sure and if I somehow fucked it up I will (remembers suicide jokes are neither healthy for myself nor those around me) try to limit the actions i take late at night when I'm not at my desktop.

SnappingOpossum replied:
I think the total count here is right but the service count is wrong

Apricite replied:
yeah, we were both wrong :>

arenaconspiracy reviewed (commented):
Remind me tomorrow when I'm on my desktop to look at the shuttle file, to see if it has the necessary DeviceNetworkComponent, it dawns on me that I forgot to check.

arenaconspiracy committed:
Update the job count summary to not lie

arenaconspiracy commented on Resources/Prototypes/_Carpmosia/Maps/feint.yml:
```diff
@@ -0,0 +1,65 @@
+          ChiefEngineer: [ 1, 1 ]
+          ResearchDirector: [ 1, 1 ]
+          Quartermaster: [ 1, 1 ]
+          # service (11 - 14)
```
```suggestion
          # service (12 - 15)
```

arenaconspiracy commented on Resources/Prototypes/_Carpmosia/Maps/feint.yml:
```diff
@@ -0,0 +1,65 @@
+      - type: StationEmergencyShuttle
+        emergencyShuttlePath: /Maps/_Carpmosia/Shuttles/emergency_feint.yml
+      - type: StationJobs
+        availableJobs: # 43 jobs total w/o latejoins & interns, 58 jobs total w/ latejoins & interns
```
```suggestion
        availableJobs: # 44 jobs total w/o latejoins & interns, 59 jobs total w/ latejoins & interns
```
hhhhhhhhhhhhhhhhhhhhhh

arenaconspiracy reviewed (approved):
Ok I counted EVERYTHING this time.

arenaconspiracy committed:
The lying liar

Merrokitsune reviewed (changes requested):
Hello. Are those trees in the southwest maints botany stray, or are they the result of a glitch with the map renderer?

Apricite commented:
I don't think I understand the question. The trees in maints botany are intentional, because it's also a mini garden

Atakku commented:
<img width="211" height="234" alt="image" src="https://github.com/user-attachments/assets/4b7af233-90cd-44d0-97b4-b8fdb9930267" />
they probably mean this
and I'm pretty sure its a map renderer bug

Merrokitsune commented:
> I don't think I understand the question. The trees in maints botany are intentional, because it's also a mini garden

Yep, Attaku pointed that out for you (thanks). You might wanna try to screenshot it in-game to see if it works or not.

Apricite commented:
Yep, map renderer error. The map is outdated anyways with the old Security
<img width="4832" height="4672" alt="Feint-0" src="https://github.com/user-attachments/assets/195ba10d-62b9-4c9c-b3ac-150bdc3829ac" />

Merrokitsune reviewed (approved):
Yep, looking good.

arenaconspiracy commented:
FEINT MERGABLE

arenaconspiracy commented:
WE WIN
